app/components/order-inline-number-input.js

```diff
@@ -5,22 +5,28 @@ const { getOwner } = Ember;
 
 export default BeneficiaryInlineInput.extend({
   type: "tel",
+  orderCopy: Ember.computed.readOnly('order'),
```

Comment:
What is the use of readOnly here? Why it is needed?

Reply:
It is to avoid updating the `order` object and maintain the `order` object when are coming back to the tab.

Reply:
I don't see any manipulation of Order in the file or PR. 
I don't think this is necessary.

Reply:
When you delete value here (https://github.com/crossroads/stock.goodcity/pull/401/files#diff-f26ff51c81035b6e0e71a7ceb8f18e1dR28), and you change tab and come back and add back value with `peopleHelped` which was empty earlier (because of reference). So ,  to maintain that state and make it unchangeable it was required to make a readonly.